src/DynamoWebServer/Program.cs

```diff
@@ -32,9 +32,9 @@ public static void Main(string[] args)
 
             webSocketServer.Start();
 
-            var app = new Application();
-            app.Exit += webSocketServer.ProcessExit;
-            app.Run();
+            Process.GetCurrentProcess().Exited += webSocketServer.ProcessExited;
+
+            while (true) {}
```

Comment:
I'm sure this is temporary, do we want to do this for the server and enhance this in the next PR?